pages/components/checkout-card.tsx

```diff
@@ -142,6 +149,21 @@ export default function CheckoutCard({
     }
   }
 
+  const sendInvoicePaid = async (metricsInvoiceId: string) => {
```

Comment:
captureIncoicePaidMetric. I think that should be the naming convention for places where we are captureing data so we can locate all of them via shift+ctrl+f